include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -187,6 +187,37 @@ def XTenNN_LoadExternalConstOp: XTenNN_Op<"load_external_const", [
 
 def TosaExtension : NativeOpTrait<"TosaExtension">;
 def ElementwiseUnary : NativeOpTrait<"ElementwiseUnary">;
+class ArrayAttr<int n> : ConfinedAttr<ArrayAttr, [ArrayCount<n>]>;
+class I64DenseArrayAttr<int n> : ConfinedAttr<DenseI64ArrayAttr, [DenseArrayCount<n>]>;
+
+def XtenNN_GroupConv2dOp: XTenNN_Op<"group_conv2d", [Pure, TosaExtension]> {
+  let summary = "Grouped convolution with multiple channel outputs per layer";
+  let description = [{
+    Calculates separates convolutions with multiple channel outputs per layer. The number
+    of groups need to be higher than one.
```

Comment:
Why does the `group==1` be illegal? (I understand we may only want to use it for that, but I don't see it as a necessary limitation of the operater).

Reply:
You are right, there is no need for this restriction